src/current/v23.1/show-sessions.md

```diff
@@ -18,7 +18,7 @@ These details let you monitor the overall state of client connections and identi
 
 ## Required privileges
 
-All users can see their own currently active sessions. All users belonging to the `admin` role can view see all users' currently active sessions. To view other non-admin users' sessions, the non-admin user must have the `VIEWACTIVITY` [system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#supported-privileges) (or the legacy [`VIEWACTIVITY`]({% link {{ page.version.version }}/create-user.md %}#create-a-user-that-can-see-and-cancel-non-admin-queries-and-sessions) [role option]({% link {{ page.version.version }}/security-reference/authorization.md %}#role-options)) defined.
+All users can see their own currently active sessions. Users with the VIEWACTIVITY or VIEWACTIVITYREDACTED privilege can view see all users' currently active sessions. VIEWACTIVITYREDACTED causes constants in queries being executed by other users to be redacted.
```

Comment:
Just a formatting / links thing

Can you please change

```
Users with the VIEWACTIVITY or VIEWACTIVITYREDACTED privilege
```

to 

```
Users with the [`VIEWACTIVITY` or `VIEWACTIVITYREDACTED` privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#supported-privileges)
```

and add backticks to the second mention of `VIEWACTIVITYREDACTED` ?